Code/GraphMol/Chirality.cpp

```diff
@@ -1112,7 +1113,15 @@ void findPotentialStereoBonds(ROMol &mol, bool cleanIt) {
             // ------------------
             // get the CIP ranking of each atom if we need it:
             if (!cipDone) {
-              Chirality::assignAtomCIPRanks(mol, ranks);
+              if (!begAtom->hasProp(common_properties::_CIPRank)) {
+                Chirality::assignAtomCIPRanks(mol, ranks);
+              } else {
+                // no need to recompute if we don't need to recompute. :-)
+                for (unsigned int ai = 0; ai < mol.getNumAtoms(); ++ai) {
```

Comment:
If they call clearComptedProps it will blow out the CIP ranks, but then this will likely generate different answers because they haven't called assignStereoChemistry(). As I mention above, this is far from a perfect solution, but it's better than what is currently there